### ISSUE TITLE
Will we get any update on V1 before V2 is due?

### ISSUE THREAD

**Issue by richy0101** [usc-csci201-fall2013/simcity201#75]
I understand that its gonna be an extremely arduous process to grade individuals for v1, but is there anyway we can get some type of update, either with only team grades or something before v2, so we're not basically "blindly" turning in 50% of our grade without knowing how we can improve on our past actions?

@dwilczyn @alghanmi @crowley1958

**Comment by johngtimms** [comment]
I agree. It would be helpful to get some sort of feedback. It shouldn't be too hard for the CPs to load up a project and determine the general state of things. (Maybe in terms of a ranking relative to other teams' work?)

**Comment by alghanmi** [comment]
@richy0101 and @johngtimms, we plan to have a quicker turn-around for v1. Keep in mind that this week is crunch time for everyone so we are all working hard to achieve the same goal:

> A successful end to an interesting CS 201
